pow/src/main/java/tech/pegasys/artemis/pow/DepositContractListener.java

```diff
@@ -44,11 +53,29 @@ public DepositContractListener(EventBus eventBus, DepositContract contract) {
     subscriptionNewDeposit =
         contract
             .depositEventEventFlowable(depositEventFilter)
-            .subscribe(
-                response -> {
-                  Deposit deposit = new Deposit(response);
-                  eventBus.post(deposit);
-                });
+            .flatMap(this::convertToDeposit)
```

Comment:
Does this mean we drop the deposit if the block request fails?

Reply:
Yes, but that's the same as if something interrupts the subscriptions or the ETH1 node is down at startup - we'll just stop processing deposits until Artemis is restarted.  We need to add some error handling and retries.